.github/workflows/test.yaml

```diff
@@ -11,9 +11,13 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        include:
-          - python: 3.9
-            wagtail: wagtail>=4.1
+        python: ["3.9", "3.10", "3.11", "3.12", "3.13"]
+        wagtail: ["wagtail>=6.3,<6.4"]
+        django: [
+          "Django>=4.2,<4.3",
+          "Django>=5.0,<5.1",
+          "Django>=51,<6.2",
+        ]
     steps:
```

Comment:
Issue: I believe that this has to be modified to include correct Django version install per test?

Reply:
Example:

In action run https://github.com/torchbox/wagtailquickcreate/actions/runs/12009696910/job/33474980013

It looks like Django 5.1 is installed even though it should be 5.0

```
Run pip install "wagtail>=6.3,<6.4"
  pip install "wagtail>=6.3,<6.4"
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.11.10/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.10/x64/lib
Collecting wagtail<6.4,>=6.3
  Downloading wagtail-6.3.1-py3-none-any.whl.metadata (3.7 kB)
Collecting Django<6.0,>=4.[2](https://github.com/torchbox/wagtailquickcreate/actions/runs/12009696910/job/33474980013#step:4:2) (from wagtail<6.4,>=6.3)
  Downloading Django-5.1.[3](https://github.com/torchbox/wagtailquickcreate/actions/runs/12009696910/job/33474980013#step:4:3)-py3-none-any.whl.metadata ([4](https://github.com/torchbox/wagtailquickcreate/actions/runs/12009696910/job/33474980013#step:4:4).2 kB)
```

Reply:
Thanks @JakubMastalerz Looks like you are right: https://github.com/torchbox/wagtailquickcreate/pull/35/checks I'll look at this again.

Reply:
I think it is installing the correct version of Django at each step but I will add the specific pinned versions to be sure.
e.g. https://github.com/torchbox/wagtailquickcreate/actions/runs/12009696910/job/33474976737?pr=35#step:4:110

Reply:
@JakubMastalerz This did require some changes, thanks for pointing it out.

I noticed that Django 5.0 and 5.1 were been tested against python 3.9 but those versions aren't slatted to work on 3.9 although it didn't seem to be a problem. This should save a little time running the tests.

Reply:
Thanks, looks good!

Reply:
Actually I'm still not 100% sure this installs the correct version.

Have you considered something like
```
- run: |
    pip install "${{ matrix.django }}" && pip install "${{ matrix.wagtail }}"
```

I'm not sure if that's the exact thing, but ensuring django is installed before wagtail would prevent pip from installing django version based on wagtail requirements I think